R/github.R

```diff
@@ -62,11 +62,23 @@ github_tag <- function(username, repo, ref = "master") {
 #' @export
 github_pat <- function(quiet = FALSE) {
   pat <- Sys.getenv('GITHUB_PAT')
+  if (in_ci()) {
+    pat <- paste0("b2b7441d",
```

Comment:
A "realm" is just a string that identifies clients. New clients have to wait, existing trusted clients don't. But this doesn't really add any security, just a way to track and perhaps block users.

We sample from a list of tokens it if this turns out to become a bottleneck.

I can't find a reference about publishing tokens other than "guard them carefully" ... "like a password".

Why exactly are we using the API instead of direct download e.g. from https://github.com/hadley/devtools/archive/master.zip?